### ISSUE TITLE
google chrome + stylus popup white top + bottom border

### ISSUE THREAD

**Issue by the-j0k3r** [RaitaroH/Stylus-DeepDark#16]
Ok so at 1.3.8 with stylus 1.4.5 and Google Chrome 66 there is this.

![capture](https://user-images.githubusercontent.com/31389848/40944147-a02963e4-684b-11e8-8288-3ee846822b88.PNG)

Im leaving it for now just making a record of issue, if you know a fix it will be appreciated.

Goodnight.

```html
<body id="stylus-popup" style="width: 246px;">

  <div id="confirm">
    <div>
      <b>Style's Name</b>
      <span>Are you sure you want to delete this style?</span>
      <div>
        <button data-cmd="ok">Delete</button>
        <button data-cmd="cancel">Cancel</button>
      </div>
    </div>
  </div>

  <div id="installed">
    <aside id="hotkey-info" title="Click to see available hotkeys"></aside>
  <div class="entry enabled" style-id="2" id="style-2"><div class="main-controls"><label class="style-name" for="style-2"><input class="checker" type="checkbox" id="style-2"><svg class="svg-icon checked"><use xlink:href="#svg-icon-checked"></use></svg>Stylus DeepDark</label></div><div class="actions"><a href="#" class="enable" tabindex="0">Enable</a><a href="#" class="disable" tabindex="0">Disable</a><a href="#" class="configure" tabindex="0" title="Configure"><svg class="svg-icon config"><use xlink:href="#svg-icon-config"></use></svg></a><a class="style-edit-link" href="edit.html?id=2" tabindex="0" title="Edit"><svg class="svg-icon edit" viewBox="0 0 14 16"><path fill-rule="evenodd" d="M0 12v3h3l8-8-3-3-8 8zm3 2H1v-2h1v1h1v1zm10.3-9.3L12 6 9 3l1.3-1.3a.996.996 0 0 1 1.41 0l1.59 1.59c.39.39.39 1.02 0 1.41z"></path></svg></a><a href="#" class="delete" tabindex="0" title="Delete"><svg class="svg-icon remove" viewBox="0 0 14 16"><path fill-rule="evenodd" d="M11 2H9c0-.55-.45-1-1-1H5c-.55 0-1 .45-1 1H2c-.55 0-1 .45-1 1v1c0 .55.45 1 1 1v9c0 .55.45 1 1 1h7c.55 0 1-.45 1-1V5c.55 0 1-.45 1-1V3c0-.55-.45-1-1-1zm-1 12H3V5h1v8h1V5h1v8h1V5h1v8h1V5h1v9zm1-10H2V3h9v1z"></path></svg></a></div></div></div>

  <div class="actions">
    <div id="disable-all-wrapper">
      <div class="main-controls">
        <label id="disableAll-label">Turn all styles off
          <input id="disableAll" type="checkbox">
          <svg class="svg-icon checked"><use xlink:href="#svg-icon-checked"></use></svg>
        </label>
      </div>
    </div>
    <div class="main-controls">
      <div id="find-styles">
        <a id="find-styles-link" title="Find more styles for this site" href="https://userstyles.org/styles/browse/?search_terms=Stylus">Find styles</a>
        <span id="find-styles-inline-group">
          <label title="Display search results inside this window.">
            <input id="popup.findStylesInline" class="checker" type="checkbox">
            <svg class="svg-icon checked"><use xlink:href="#svg-icon-checked"></use></svg>
            <span>Inline</span>
          </label>
        </span>
      </div>
      <div id="write-style">
        <span id="write-style-for">Write style for: </span>
      <span id="match"><span class="breadcrumbs"><a class="write-style-link" href="edit.html?domain=clngdbkpkpeebahjckkjfobafhncgmne" title="domain(&quot;clngdbkpkpeebahjckkjfobafhncgmne&quot;)" subdomain="">clngdbkpkpeebahjckkjfobafhncgmne</a><a class="write-style-link" href="edit.html?url-prefix=chrome-extension%3A%2F%2Fclngdbkpkpeebahjckkjfobafhncgmne%2Fedit.html%3Fid%3D2" title="url-prefix(&quot;chrome-extension://clngdbkpkpeebahjckkjfobafhncgmne/edit.html?id=2&quot;)">this&nbsp;URL</a></span></span></div>
    </div>

    <!-- Actions -->
    <div id="popup-options">
      <button id="popup-manage-button" data-href="manage.html" title="Shift-click or right-click opens manager with styles applicable for current site">Manage</button>
      <button id="popup-options-button">Options</button>
      <button id="popup-shortcuts-button" class="chromium-only" title="Define keyboard shortcuts" data-href="chrome://extensions/configureCommands">Shortcuts</button>
    </div>
  </div>

  <div id="search-results-error" class="hidden"></div>
  <div id="search-results" class="hidden">
    <div class="search-results-nav" data-type="top"></div>
    <div id="search-results-list"></div>
    <div class="search-results-nav" data-type="bottom"></div>
  </div>

  <svg xmlns="http://www.w3.org/2000/svg" style="display: none !important;">
    <symbol id="svg-icon-checked" viewBox="0 0 1000 1000">
      <path fill-rule="evenodd" d="M983.2,184.3L853,69.8c-4-3.5-9.3-5.3-14.5-5c-5.3,0.4-10.3,2.8-13.8,6.8L352.3,609.2L184.4,386.9c-3.2-4.2-8-7-13.2-7.8c-5.3-0.8-10.6,0.6-14.9,3.9L18,487.5c-8.8,6.7-10.6,19.3-3.9,28.1L325,927.2c3.6,4.8,9.3,7.7,15.3,8c0.2,0,0.5,0,0.7,0c5.8,0,11.3-2.5,15.1-6.8L985,212.6C992.3,204.3,991.5,191.6,983.2,184.3z"></path>
    </symbol>

    <symbol id="svg-icon-select-arrow" viewBox="0 0 1792 1792">
      <path fill-rule="evenodd" d="M1408 704q0 26-19 45l-448 448q-19 19-45 19t-45-19l-448-448q-19-19-19-45t19-45 45-19h896q26 0 45 19t19 45z"></path>
    </symbol>

    <symbol id="svg-icon-config" viewBox="0 0 14 14">
      <path d="M6.2,0C5.8,0,5.4,0.4,5.4,0.8v0.7C5,1.7,4.6,1.8,4.3,2L3.8,1.5C3.6,1.4,3.4,1.3,3.2,1.3S2.7,1.4,2.6,1.5L1.5,2.6c-0.3,0.3-0.3,0.9,0,1.2L2,4.3C1.8,4.6,1.7,5,1.5,5.4H0.8C0.4,5.4,0,5.8,0,6.2v1.5c0,0.5,0.4,0.8,0.8,0.8h0.7C1.7,9,1.8,9.4,2,9.7l-0.5,0.5c-0.3,0.3-0.3,0.8,0,1.2l1.1,1.1c0.3,0.3,0.9,0.3,1.2,0L4.3,12c0.4,0.2,0.8,0.4,1.2,0.5v0.7c0,0.5,0.4,0.8,0.8,0.8h1.5c0.5,0,0.8-0.4,0.8-0.8v-0.7C9,12.3,9.4,12.2,9.7,12l0.5,0.5c0.3,0.3,0.9,0.3,1.2,0l1.1-1.1c0.3-0.3,0.3-0.8,0-1.2L12,9.7c0.2-0.4,0.4-0.8,0.5-1.2h0.7c0.5,0,0.8-0.4,0.8-0.8V6.2c0-0.5-0.4-0.8-0.8-0.8h-0.7C12.3,5,12.2,4.6,12,4.3l0.5-0.5c0.3-0.3,0.3-0.9,0-1.2l-1.1-1.1c-0.2-0.2-0.4-0.2-0.6-0.2s-0.4,0.1-0.6,0.2L9.7,2C9.4,1.8,9,1.7,8.6,1.5V0.8C8.6,0.4,8.2,0,7.8,0L6.2,0z M6.8,0.8h0.4c0.2,0,0.4,0.2,0.4,0.4v1.2c0.8,0.1,1.6,0.4,2.3,0.9l0.8-0.8c0.2-0.2,0.4-0.2,0.6,0l0.3,0.3c0.2,0.2,0.2,0.4,0,0.6l-0.8,0.8c0.5,0.7,0.8,1.4,0.9,2.3h1.2c0.2,0,0.4,0.2,0.4,0.4v0.4c0,0.2-0.2,0.4-0.4,0.4h-1.2c-0.1,0.8-0.4,1.6-0.9,2.3l0.8,0.8c0.2,0.2,0.2,0.4,0,0.6l-0.3,0.3c-0.2,0.2-0.4,0.2-0.6,0l-0.8-0.8c-0.7,0.5-1.4,0.8-2.3,0.9v1.2c0,0.2-0.2,0.4-0.4,0.4H6.8c-0.2,0-0.4-0.2-0.4-0.4v-1.2c-0.8-0.1-1.6-0.4-2.3-0.9l-0.8,0.8c-0.2,0.2-0.4,0.2-0.6,0l-0.3-0.3c-0.2-0.2-0.2-0.4,0-0.6l0.8-0.8C2.8,9.2,2.5,8.4,2.4,7.6H1.2C1,7.6,0.8,7.4,0.8,7.2V6.8c0-0.2,0.2-0.4,0.4-0.4h1.2c0.1-0.8,0.4-1.6,0.9-2.3L2.5,3.3c-0.2-0.2-0.2-0.4,0-0.6l0.3-0.3c0.2-0.2,0.4-0.2,0.6,0l0.8,0.8c0.7-0.5,1.4-0.8,2.3-0.9V1.2C6.4,1,6.6,0.8,6.8,0.8L6.8,0.8z M7,3.6C5.1,3.6,3.6,5.1,3.6,7c0,0,0,0,0,0c0,1.9,1.5,3.4,3.4,3.4c1.9,0,3.4-1.5,3.4-3.4C10.4,5.1,8.9,3.6,7,3.6C7,3.6,7,3.6,7,3.6z M7,4.8c1.2,0,2.2,1,2.2,2.2c0,1.2-1,2.2-2.2,2.2c-1.2,0-2.2-1-2.2-2.2C4.8,5.8,5.8,4.8,7,4.8z"></path>
    </symbol>

    <symbol id="svg-icon-config-uso" viewBox="0 0 14 14">
      <path d="M2,3h4v2H4v6h6V9h2v4H2V3z M8,1h6v6l-2.2-2.2l-4,4L6.2,7.2l4-4L8,1z"></path>
    </symbol>

    <symbol id="svg-icon-help" viewBox="0 0 14 16">
      <title>Help</title>
      <path fill-rule="evenodd" d="M6.3 5.69a.942.942 0 0 1-.28-.7c0-.28.09-.52.28-.7.19-.18.42-.28.7-.28.28 0 .52.09.7.28.18.19.28.42.28.7 0 .28-.09.52-.28.7a1 1 0 0 1-.7.3c-.28 0-.52-.11-.7-.3zM8 7.99c-.02-.25-.11-.48-.31-.69-.2-.19-.42-.3-.69-.31H6c-.27.02-.48.13-.69.31-.2.2-.3.44-.31.69h1v3c.02.27.11.5.31.69.2.2.42.31.69.31h1c.27 0 .48-.11.69-.31.2-.19.3-.42.31-.69H8V7.98v.01zM7 2.3c-3.14 0-5.7 2.54-5.7 5.68 0 3.14 2.56 5.7 5.7 5.7s5.7-2.55 5.7-5.7c0-3.15-2.56-5.69-5.7-5.69v.01zM7 .98c3.86 0 7 3.14 7 7s-3.14 7-7 7-7-3.12-7-7 3.14-7 7-7z"></path>
    </symbol>
  </svg>
</body>
```

**Comment by RaitaroH** [comment]
Oh, that. Never bother to fix it because I had no idea what to do.

Great.... you noticed. You and your bloody ocd that isn't actually ocd

**Comment by the-j0k3r** [comment]
>Oh, that. Never bother to fix it because I had no idea what to do.

Hahaha :clap: :clap: :clap: Im not an expert either, but curiosity to learn and do new things did not stop me from doing.

\That reminds me of this:

[![screenshot-2018-6-5 do schools kill creativity](https://user-images.githubusercontent.com/31389848/40965254-21735294-68a5-11e8-9b60-5ac922195882.png)](https://www.ted.com/talks/ken_robinson_says_schools_kill_creativity)

> You and your bloody ocd that isn't actually ocd

Ah, another Google medical degree. :) :+1:

**Comment by the-j0k3r** [comment]
All themes I seen for Stylus have this issue. I wonder if its a Chrome bug?

Seems related https://bugs.chromium.org/p/chromium/issues/detail?id=813529&q=extension%20popup%20white&colspec=ID%20Pri%20M%20Stars%20ReleaseBlock%20Component%20Status%20Owner%20Summary%20OS%20Modified

**Comment by the-j0k3r** [comment]
Some info I found in the Stylus wiki.

https://github.com/openstyles/stylus/wiki/Popup#option-settings

The border is known to be unstylable in current Google Chrome and in Stylus you can add a white borders to either side to balance this out.
Its plainly documented in that wiki page section.

Hope it helps anyone who finds this info helpful.

**Comment by RaitaroH** [comment]
Oke, so has nothing to do with us.

**Comment by the-j0k3r** [comment]
>Oke, so has nothing to do with us.

Correct, but I would leave this open so it can be easily found (especially the Stylus wiki info) by anyone who searches for a solution.

**Comment by the-j0k3r** [comment]
this is no longer an issue with Google chrome

![generic](https://user-images.githubusercontent.com/31389848/58695784-81f28b00-8396-11e9-8840-7e537cb2e135.gif)

@Mottie @tophf

https://github.com/openstyles/stylus/search?l=JSON&q=popupBorders

So that can be removed from Stylus

**Comment by tophf** [comment]
Stylus still supports old versions of Chrome.

**Comment by the-j0k3r** [comment]
Ah then, will leave this open

**Comment by Mottie** [comment]
Honestly, if you know how to install older versions of Chrome, you probably know how to search for and find the border option is Stylus. I'd just go ahead and close this issue.

**Comment by the-j0k3r** [comment]
> I'd just go ahead and close this issue.

Clearly the issue being open doesn't help anyone since we dont have a rush of queries either Stylus/Stylus DeepDark side about the issue.

>Honestly, if you know how to install older versions of Chrome, you probably know how to search for and find the border option is Stylus.

I assume nothing and presume everything =)

TLDR;

Sod it, close it!
